frontend/www/js/omegaup/components/problem/Details.vue

```diff
@@ -251,12 +262,14 @@ export default class ProblemDetails extends Vue {
   @Prop({ default: 0 }) availableTokens!: number;
   @Prop({ default: 0 }) allTokens!: number;
   @Prop() histogram!: types.Histogram;
+  @Prop() initialTab!: string;
 
   T = T;
   ui = ui;
   time = time;
-  selectedTab = 'problems';
+  selectedTab = this.initialTab;
   clarifications = this.initialClarifications || [];
+  clarificationsTabVisited = false;
```

Comment:
esto debería de regresarse a `false` en algún punto. si no, si alguien visita la pestaña de clarificaciones, jamás se va a volver a desplegar en bold.

una mejor solución es cambiarlo a


```suggestion
  hasUnreadClarifications = this.initialClarifications.length > 0 && this.activeTab != 'clarifications';
```
y ponerle un Watcher a `clarifications` para volverlo verdadero cuando se cambie (siempre y cuando `selectedTab != 'clarifications'`).

Reply:
Listo.